README.md

```diff
@@ -315,17 +308,28 @@ that it can be deployed in. There are generally two ways to get trace data
 out of the system.
 
 The first is to use an I/O interface on your device and use the `report` API to
-send data to a waiting collector, like in this UDP oriented example below:
-
-```rust
-fn send_report<'a>(socket: &UdpSocket, probe: &mut ModalityProbe<'a>, report_buffer: &mut [u8]) {
-    let n_report_bytes = probe
-        .report(report_buffer)
-        .expect("Could not produce a report");
-    if let Some(nonzero_report_size) = n_report_bytes {
-        socket
-            .send_to(&report_buffer[..nonzero_report_size.get()], COLLECTOR_ADDR)
-            .expect("Could not send_to");
+send data to a waiting collector, like in this UDP-oriented example below:
+
+```c
+{
```

Comment:
Appears to be missing the beginning part of the function definition from the old capi snippet:
> static void send_report(modality_probe * const probe)
{
...

Reply:
:+1: Great catch, thanks!